extensions/wikia/GlobalShortcuts/GlobalShortcutsHelper.php

```diff
@@ -81,9 +81,25 @@ public function addSpecialPageActions( array &$actions, array &$shortcuts ) {
 			}
 		}
 
+		$specialPagesMsg = wfMessage( 'specialpages' )->plain();
```

Comment:
This message isn't being escaped later (none in the modal are). Either modal should make sure to escape the text of the shortcuts, or they should be escaped before being passed to the modal. Same for the groupName messages below and possibly others.